1-Vanilla/index.js

```diff
@@ -0,0 +1,172 @@
+let editedID = 0;
+let editMode = false;
+
+function getNewID(){
+    return Math.floor(Math.random()*1000)
+}
+function addItem(){
+    let title = document.getElementById('title-input').value
```

Comment:
the default should be always `const` unless your'e going to change either the pointer or if it's a primitive its value.
change all `let` to `const` unless u have to use `let`